ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -399,29 +399,8 @@ QtObject {
         return d.mainModuleInst.addressWasShown(address)
     }
 
-    function getExplorerDomain(networkShortName) {
```

Comment:
This is still used in CollectibleDetailView.qml, as such:
```qml
onOpenCollectibleOnExplorer: Global.openLinkWithConfirmation(d.blockExplorerLink, root.walletRootStore.getExplorerDomain(networkShortName))
```

Reply:
ooops!
the code was basically duplicated in Utils but I didn't properly check all usages of this one, will re-check. Thanks!